src/lib/REST/Value/ContentType/FieldDefinitionCreate.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\REST\Value\ContentType;
+
+use EzSystems\EzPlatformRest\Value as RestValue;
+
+final class FieldDefinitionCreate extends RestValue
+{
+    /** @var string|null */
+    public $fieldTypeIdentifier;
+
+    /** @var string|null */
+    public $fieldGroupIdentifier;
+
+    /** @var int|null */
+    public $position;
+
+    public function __construct(?string $fieldTypeIdentifier, ?string $fieldGroupIdentifier, ?int $position = null)
```

Comment:
```suggestion
    public function __construct(?string $fieldTypeIdentifier = null, ?string $fieldGroupIdentifier = null, ?int $position = null)
```

Reply:
@mikadamczyk in this case I believe it's expected that this argument is always explicitly passed, even if `null`.

Reply:
@mikadamczyk yeah, it always has a value passed to it. I'd prefer to be explicit here, since we're talking about field type identifier & field group identifier.